modules/log-collector-http-server.adoc

```diff
@@ -23,7 +23,7 @@ You can configure your log collector to listen for HTTP connections and receive
 .Example `ClusterLogForwarder` CR if you are using a multi log forwarder deployment
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1beta1
+apiVersion: logging.openshift.io/v1
```

Comment:
@theashiot   Is there a issue about how to enable hypershift hosted master to send logs to this collector httpserver?  If not, I can file one.

Reply:
Ok. let me file one.